src/Collectors/JobCollector.php

```diff
@@ -41,6 +41,7 @@ public function registerEventListeners(): void
 
             $this->startTransaction($transaction_name);
             $this->setTransactionType($transaction_name);
+            $this->addMetadata($transaction_name, $event->job);
```

Comment:
Next iteration something like this to allow users to set their own metadata via a config-defined callback:

```php
if (Config::get('elastic-apm-laravel.transactions.metadata_provider')) {
    $this->addMetadata($transaction_name, call_user_func(Config::get('elastic-apm-laravel.transactions.metadata_provider'), $event));
}
```